pkgs/by-name/ri/rido/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  openssl,
+  pkg-config,
```

Comment:
Could also patch it to use openSSL instead, basically reverting this commit
https://github.com/lj3954/rido/commit/a1aa36a6108b34d2495256868b3b720170b8c306
```suggestion
```

Reply:
I've seen this commit too. But, I would prefer to stay with RusTLS here...